ServerCore/Pages/Pieces/InternMeta.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@page "/{eventId}/{eventRole}/Teams/{teamId}/InternMeta/{puzzleId}"
```

Comment:
What does the puzzleId refer to here? Is is supposed to be the puzzleId of the meta?

Reply:
Yes, this is how the piece data works